src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
 }
+
+@import './blocks/header';
```

Comment:
Imports should be placed at the beginning of the file. It's a good practice to keep all your import statements grouped together at the top of your SCSS file before any actual styles are written.